examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,58 +14,56 @@
   },
```

Comment:
<p>Try </p><pre class="ql-syntax" spellcheck="false">az.plot_dist(trace.sample_stats["acceptance_rate"])
</pre><p> </p><p>and also </p><p> </p><pre class="ql-syntax" spellcheck="false">az.plot_posterior(trace, group="sample_stats", var_names="acceptance_rate", hdi_prob="hide", kind="hist")
</pre><p> </p><p>which will also report the mean on the same plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>